CMSgov/easi-app
# EASI-3786/remove v1 actions

# EASI-3786

## Changes and Description

- remove V1 admin actions based on: https://publish.obsidian.md/clay-docs/Work/Research+for+EASI-3493
- removed anything related to the deprecated V1 admin actions (resolvers and service code + tests where applicable)

## How to test this change



- Pull branch and confirm all tests pass
- Run branch locally, attempt all admin actions (extending LCID, Rejecting an intake, adding GRT feedback, etc) where still applicable, and confirm successful action

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
